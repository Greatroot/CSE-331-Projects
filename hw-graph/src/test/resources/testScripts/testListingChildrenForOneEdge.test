# Tests listing children for a node with one edge | Should return that one child

CreateGraph graph
AddNode graph n1
AddNode graph n2
AddEdge graph n1 n2 e1
ListChildren graph n1