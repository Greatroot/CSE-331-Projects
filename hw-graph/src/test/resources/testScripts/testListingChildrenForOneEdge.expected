# Tests listing children for a node with one edge | Should return that one child

created graph graph
added node n1 to graph
added node n2 to graph
added edge e1 from n1 to n2 in graph
the children of n1 in graph are: n2(e1)